Rubberduck.Resources/Settings/SettingsUI.resx

```diff
@@ -199,7 +199,7 @@
     <value>Configure which autocompletions are enabled.</value>
   </data>
   <data name="PageInstructions_GeneralSettings" xml:space="preserve">
-    <value>Click [Ok] to close the window and apply changes, or [Cancel] to discard them.</value>
+    <value>Click [{0}] to close the window and apply changes, or [{1}] to discard them.</value>
```

Comment:
I like the implementation but I am wondering why would we need a "click x to do y" instruction that's not on the other pages. I mean, "click x to do y" is true for all pages, isn't it? So why are we calling it out only for the general page? Seems to me that the general page should be just "Configure how Rubberduck should behave, whether it should modify VBIDE and what hotkeys to use", which is more in line with other page instructions.

Reply:
This was a direct update addressing the new text. I didn't think more than than when doing the update. Thinking it through [Instructions](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.Core/UI/Settings/SettingsView.cs#L9) should be updated to a more semantic wording like `Description`.